xblockutils/public/studio_edit.js

```diff
@@ -65,6 +66,11 @@ function StudioEditableXBlockMixin(runtime, element) {
                 }
             });
         }
+
+        if (type == 'datepicker' && datepickerAvailable) {
+            $field.datepicker('destroy');
+            $field.datepicker({dateFormat: "m/d/yy"});
```

Comment:
Hardcoding the format is probably not a good idea as it won’t work for localized Open edX instances.